### PR TITLE
Updates to Swift 5 and Xcode 10.2

### PR DESCRIPTION
**Pull request by eSpecialized**

### CODE REVIEW

File: SQift.podspec [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+  s.version = "4.1.0"
   s.license = "MIT"
   s.summary = "A lightweight Swift wrapper for SQLite."
   s.homepage = "https://github.com/Nike-Inc/SQift"
   s.authors = { "Dave Camp" => "dave.camp@nike.com", "Christian Noon" => "christian.noon@nike.com" }
 
   s.source = { :git => "https://github.com/Nike-Inc/SQift.git", :tag => s.version }
   s.source_files = "Source/**/*.swift"
-  s.swift_version = "4.2"
+  s.swift_versions = ['4.2', '5.0']
```
Comment (cnoon) [inline SQift.podspec]: Could we swap single quotes out for double quotes?

Comment (eSpecialized) [reply SQift.podspec]: Done

File: Gemfile [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+#
+#  Created by Christian Noon on 6/1/18.
+#  Copyright © 2018 Nike. All rights reserved.
+#
+
 source "https://rubygems.org"
 
-gem "cocoapods", "~> 1.6.0.beta.1"
 gem "xcpretty"
+gem "cocoapods", "~> 1.7.0.beta.3"
```
[the remark this answers is not in the document]
Comment (eSpecialized) [reply Gemfile]: Done

File: Gemfile [hunk 3]
```diff
@@ -1,4 +1,13 @@
+#
+#  Gemfile
+#  BuildTools
+#
+#  Created by Christian Noon on 6/1/18.
+#  Copyright © 2018 Nike. All rights reserved.
```
[the remark this answers is not in the document]
Comment (eSpecialized) [reply Gemfile]: Done

File: Gemfile [hunk 4]
```diff
@@ -1,4 +1,13 @@
+#
+#  Gemfile
+#  BuildTools
```
[the remark this answers is not in the document]
Comment (eSpecialized) [reply Gemfile]: Updated

File: CHANGELOG.md [hunk 5]
```diff
@@ -36,12 +37,23 @@ All notable changes to this project will be documented in this file.
...
 
+---
+## [4.1.0](https://github.com/Nike-Inc/SQift/releases/tag/4.1.0)
+
+Release on 2019-04-30.  All issues associated with this milestone can be found using this
+[filter](https://github.com/Nike-Inc/SQift/milestone/8?closed=1).
+
+#### Updated
+-  To Swift 5 with backwards compatibility to Swift 4.2.
+-  Bumped Deployment targets to Watch OS 3.0 and TVos 10.0
```
[the remark this answers is not in the document]
Comment (eSpecialized) [reply CHANGELOG.md]: ok

File: CHANGELOG.md [hunk 6]
```diff
@@ -36,12 +37,23 @@ All notable changes to this project will be documented in this file.
...
 - `0.1.x` Releases - [0.1.0](#010)
 
+---
+## [4.1.0](https://github.com/Nike-Inc/SQift/releases/tag/4.1.0)
+
+Release on 2019-04-30.  All issues associated with this milestone can be found using this
+[filter](https://github.com/Nike-Inc/SQift/milestone/8?closed=1).
+
+#### Updated
+-  To Swift 5 with backwards compatibility to Swift 4.2.
```
[the remark this answers is not in the document]
Comment (eSpecialized) [reply CHANGELOG.md]: done

File: CHANGELOG.md [hunk 7]
```diff
@@ -36,12 +37,23 @@ All notable changes to this project will be documented in this file.
...
+-  Bumped Deployment targets to Watch OS 3.0 and TVos 10.0
+ - Updated by [William Thompson](https://github.com/eSpecialized) in Pull Request [#18](https://github.com/Nike-Inc/SQift/pull/18)
```
[the remark this answers is not in the document]
Comment (eSpecialized) [reply CHANGELOG.md]: done

File: Gemfile [hunk 8]
```diff
@@ -1,4 +1,13 @@
+#
+#  Gemfile
+#  SQift
+#
+#  Created by Christian Noon on 6/1/18.
```
Comment (cnoon) [inline Gemfile]: Whoops, could you update this as well?

**Pull request merged**